tests/e2e/util.go

```diff
@@ -1772,7 +1879,8 @@ func GetPodSpecByUserID(ns string, nodeSelector map[string]string, pvclaims []*v
 	for index, pvclaim := range pvclaims {
 		volumename := fmt.Sprintf("volume%v", index+1)
 		volumeMounts[index] = v1.VolumeMount{Name: volumename, MountPath: "/mnt/" + volumename}
-		volumes[index] = v1.Volume{Name: volumename, VolumeSource: v1.VolumeSource{PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{ClaimName: pvclaim.Name, ReadOnly: false}}}
+		volumes[index] = v1.Volume{Name: volumename, VolumeSource: v1.VolumeSource{
+			PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{ClaimName: pvclaim.Name, ReadOnly: false}}}
```

Comment:
```golang
		volumes[index] = v1.Volume{Name: volumename,
			VolumeSource: v1.VolumeSource{
				PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{
					ClaimName: pvclaim.Name, ReadOnly: false,
		}}}
```
might be better

Reply:
Thanks for the review and suggestion, @sashrith.

Would you approve this change? Also, would you like to use a follow-up change to fix the pattern you proposed? I like it. Appreciated.

Reply:
Thanks @sashrith for following up.